### ISSUE TITLE
removes commas in loop "for"

### ISSUE THREAD

**Issue by x404** [ConnectThink/WP-SCSS#161]
example:
```
@for $i from 1 through 12{
	.grid#{$infix}-#{$i}{
		grid-template-columns: repeat(#{$i}, 1fr);
	}
}
```

converts to

```
.grid-1 {
  grid-template-columns: repeat(11fr);
}
.grid-2 {
  grid-template-columns: repeat(21fr);
}
```
etc.

Instead
```
.grid-1 {
  grid-template-columns: repeat(1, 1fr);
}
.grid-2 {
  grid-template-columns: repeat(2, 1fr);
}
```

**Comment by shadoath** [comment]
@x404 This is probably related to [ScssPhp](https://github.com/scssphp/scssphp/releases), can you find out if it was patched there and if so what version? Currently WP-SCSS uses 1.0.3

**Comment by x404** [comment]
I have reinstalled the plugin from the wordpress repository again and in the wp-content/plugins/wp-scss/scssphp/src/Version.php file this string
```
class Version
{
    const VERSION = 'v1.0.2';
}
```
This is the most recent version in the repository WP

**Comment by x404** [comment]
I replaced all the files from the GIT repository and everything worked correctly. I'm happy.
It turns out that the repository WP contains an incorrectly working version of plugin.

**Comment by shadoath** [comment]
@x404 Thanks for pointing that out. I must have missed some files when using SVN to update the plugin. This has been corrected in version 2.1.1
